src/main/java/com/github/tomakehurst/wiremock/core/WireMockApp.java

```diff
@@ -129,45 +129,13 @@ public WireMockApp(
     }
 
     public AdminRequestHandler buildAdminRequestHandler() {
-        AdminRoutes adminRoutes = AdminRoutes.defaultsPlus(
-            options.extensionsOfType(AdminApiExtension.class).values(),
-            options.getNotMatchedRenderer()
-        );
-        return new AdminRequestHandler(
-            adminRoutes,
-            this,
-            new BasicResponseRenderer(),
-            options.getAdminAuthenticator(),
-            options.getHttpsRequiredForAdminApi(),
-            getAdminRequestFilters()
-        );
+        return new AdminRequestHandlerFactory(). // TODO: field? Static?
```

Comment:
I guess this would go into `Options`, following how we've done it in `WireMockServer`